atspi/examples/currently-focused-frame.rs

```diff
@@ -26,11 +26,19 @@ async fn main() -> Result<(), Box<dyn Error>> {
 	let mut found_active_frame: bool = false;
 
 	for app in apps.iter() {
+		if app.is_null() {
```

Comment:
Similarly here. From the outside (of atspi land) this looks like nonsense. I can see the confusion now: "Why would the application registry send back a null reference?"